src/main/java/net/sf/jabref/exporter/FileActions.java

```diff
@@ -527,7 +525,7 @@ public static Reader getReader(String name) throws IOException {
         if (keySet != null) {
```

Comment:
Can one not just iterater over the keySet using foreach/extended for syntax sugar?